### ISSUE TITLE
Add prerequisites to morning challenges and workshops

### ISSUE THREAD

**Issue by des-des** [foundersandcoders/master-reference#175]
We should be clear on what we expect students to know for a particular task. This will also help with future curriculum planning.

**Comment by idantb101** [comment]
Would you like to expand this to all the curriculum?

**Comment by des-des** [comment]
@idantb101 agreed.
I think first we need all the workshops etc with pre-reqs. Once we have this, then we can go through the weeks and pull the pre-reqs up into the top level (ie a week). That should then cover everything

**Comment by idantb101** [comment]
Any action points? @des-des

**Comment by des-des** [comment]
@idantb101 I am no longer maintaining here. Linking in @eliascodes @m4v15

**Comment by zurda** [comment]
Personally, it feels that if the workshops are scheduled in a specific week then the previous weeks probably include the prerequisites. Unless I'm missing something, and this is outdated. 
IF this is an ongoing discussion or WIP - I'm happy to help with a PR

**Comment by m4v15** [comment]
Probably outdated? Although I actually do really like this idea tbh....

**Comment by astroash** [comment]
Yeah, this is a great idea. It helps frame learning for students as it highlights which prior knowledge they will be using in that ws. 

I think newer workshops were created with this, but I’m sure it’s missing in a bunch. It would be great if we could go through all currently used FAC workshops and create issues/PRs to add this in. @zurda is this something you would be interested in doing?
